### ISSUE TITLE
Add option to ignore all day events

### ISSUE THREAD

**Issue by nikolowry** [artisticat1/gnome-next-up#14]
Please add the ability to ignore all day events. It's confusing when there are holidays and birthdays.

Attached screenshot of the experience on 2023.06.14 which is an American federal holiday named Flag Day

![Screenshot from 2023-06-14 09-36-47](https://github.com/artisticat1/gnome-next-up/assets/597310/050b57ea-453d-4a8d-bccc-9b22858b29ac)

**Comment by krillin666** [comment]
Is there any intention of implementation of this feature ?

Maybe see how https://github.com/chmouel/gnome-next-meeting-applet does it ?

**Comment by coignard** [comment]
> Is there any intention of implementation of this feature ?
> 
> Maybe see how https://github.com/chmouel/gnome-next-meeting-applet does it ?

```js
function getTodaysEvents(calendarSource) {

    const src = calendarSource;
    src._loadEvents(true);

    const today = new Date();
    today.setHours(0, 0, 0, 0); // Get event from today at midnight

    const tomorrow = new Date(today);
    tomorrow.setDate(today.getDate() + 1);

    let todaysEvents = src.getEvents(today, tomorrow);

    todaysEvents = todaysEvents.filter(event => {
        const start = event.date;
        const end = event.end;
        return !(start.getHours() === 0 && start.getMinutes() === 0 &&
                 end.getHours() === 0 && end.getMinutes() === 0);
    });

    return todaysEvents;
}
```

**Comment by muerwre** [comment]
Thanks to @coignard, implemented and tested proposed fix in PR #19. 

Hope, @artisticat1 will review and accept this.
